lib/stack.js

```diff
@@ -1,19 +1,20 @@
+//first in first out
+
 class Stack {
```

Comment:
👀 While your implementation of the Stack does work, the instructions ask you to use a Linked List to implement the stack!